tests/manage/z_cluster/test_add_capacity_flexible_scaling.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_bm,
+)
+from ocs_ci.framework.testlib import (
+    ignore_leftovers,
+    ManageTest,
+    tier1,
+    acceptance,
+    flexible_scaling_required,
+)
+from ocs_ci.utility.localstorage import check_pvs_created
+from ocs_ci.ocs.node import add_disk_to_node
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_osd_pods
+from ocs_ci.ocs.cluster import check_ceph_health_after_add_capacity
+from ocs_ci.ocs.resources.storage_cluster import (
+    osd_encryption_verification,
+    get_storage_cluster,
+)
+from ocs_ci.framework.pytest_customization.marks import skipif_managed_service
+from ocs_ci.ocs.ui.helpers_ui import ui_add_capacity_conditions
+from ocs_ci.ocs.node import get_nodes
+from ocs_ci.ocs.ui.base_ui import login_ui, close_browser
+from ocs_ci.ocs.ui.add_replace_device_ui import AddReplaceDeviceUI
+
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier1
+@acceptance
+@skipif_bm
+@flexible_scaling_required
+@skipif_managed_service
+class TestAddCapacityFlexibleScaling(ManageTest):
```

Comment:
the current add-capacity test can also run on LSO, it will be repetitive to run the test twice. Please discuss on the next steps and make sure you handle it